src/Security/MemberAuthenticator/CMSMemberLoginForm.php

```diff
@@ -58,15 +58,21 @@ public function getFormFields()
                 'Password',
                 CheckboxField::create(
                     "Remember",
-                    _t('SilverStripe\\Security\\Member.KEEPMESIGNEDIN', "Keep me signed in")
-                )->setAttribute(
-                    'title',
                     _t(
-                        'SilverStripe\\Security\\Member.REMEMBERME',
-                        "Remember me next time? (for {count} days on this device)",
+                        'SilverStripe\\Security\\Member.REMEMBER_ME',
+                        "Remember me for {count} days",
                         [ 'count' => RememberLoginHash::config()->uninherited('token_expiry_days') ]
                     )
                 )
+                    ->setAttribute('data-toggle', 'tooltip')
```

Comment:
```suggestion
                    ->setAttribute('data-toggle', 'tooltip')
```

The data-toggle doesn't really add anything, we can simply do this in login-forms:

`<i class="font-icon-help-circled" title="$getAttribute('title')" data-toggle="$getAttribute('data-toggle')">`
to
`<i class="font-icon-help-circled" title="$getAttribute('title')" data-toggle="tooltip"></i>`